packages/host-metrics/src/BaseMetrics.ts

```diff
@@ -27,6 +27,7 @@ export interface MetricsCollectorConfig {
   meterProvider?: MeterProvider;
   // Name of component
   name?: string;
+  metricGroups?: Array<string>;
```

Comment:
Style nit-picking, this can be shorter with braket types:

```suggestion
  metricGroups?: string[];
```